commands/http/client.go

```diff
@@ -117,6 +115,8 @@ func (c *client) Send(req cmds.Request) (cmds.Response, error) {
 		// using the overridden JSON encoding in request
 		res, err := getResponse(httpRes, req)
 		if err != nil {
+			//log.Error("ERROR ERROR ERROR")
+			httpRes.Body.Close()
```

Comment:
Race condition with tr.CancelRequest: this doesn't get called because `c.httpClient.Do` exits with err.

I think this is the underlying cause.